### PR TITLE
cowsay: simplify example

### PR DESCRIPTION
**Pull request by mebeim**
No need for an `echo` + pipe for the third example.

### CODE REVIEW

**Comment by agnivade** [comment]
I think the intention was to show that cowsay accepts text from the standard input also. Because the previous examples already show text as a third parameter. ping @fluxw42 (the original author of the page)

**Comment by mebeim** [comment]
@agnivade Oh, I see. I got confused by this one because I thought the only way to use another character was to use stdin. If that's the case I think it would be better to add another example. I could add it to this PR, what do you think?

**Comment by agnivade** [comment]
Sure, SGTM.

**Comment by mebeim** [comment]
Done. I think this should be rebased rather than squashed to avoid confusion with the commit message.

File: pages/common/cowsay.md [hunk 1]
```diff
@@ -18,3 +18,7 @@
 - Print a stoned thinking ASCII cow:
 
 `cowthink -s "I'm just a cow, not a great thinker ..."`
+
+- Use text from standard input for the balloon:
```
Comment (agnivade) [inline pages/common/cowsay.md]: I think you can make this the 2nd example.

Comment (mebeim) [reply pages/common/cowsay.md]: I was wondering about moving it up... will do :)

Comment (agnivade) [reply pages/common/cowsay.md]: @mebeim - Did you commit your changes ? I don't see them.

Comment (mebeim) [reply pages/common/cowsay.md]: @agnivade oh sorry, I committed without adding any change :laughing: should be ok now.

**Pull request merged**